kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/richmessaging/RichMessageActionProcessor.java

```diff
@@ -161,7 +170,9 @@ public void handleQuickReplies(Object object, Map<String, Object> replyMetadata)
             }
         } else if (object instanceof ALRichMessageModel.AlAction) {
             ALRichMessageModel.AlAction action = (ALRichMessageModel.AlAction) object;
+            updateLanguage(action.getUpdateLanguage());
```

Comment:
We can move this `updateLanguage`  method to the bottom of this `else if` and store the language in the string  and use it